### PR TITLE
[MNT] Faster docstring examples - `ForecastingGridSearchCV`, `MultiplexForecaster`

### PR DESCRIPTION
**Pull request by fkiraly**
This PR speeds up the two slowest docstring examples by reducing the size of data sets in those examples:
`ForecastingGridSearchCV` and `MultiplexForecaster`. This saves ca 60-90sec in testing (according to CI/CD and local timings).

### CODE REVIEW

**Comment by aiwalter** [comment]
I would rather remove the `AutoARIMA` and maybe replace with another fast model such as `STLForecaster`. Maybe also make `step_length` bigger. Just trimming the data looks a bit odd. Other solution might be to use `load_shampoo_sales` which has length 36

**Comment by fkiraly** [comment]
> I would rather remove

I don't think that would make a difference, but if you don't like it, feel free to replace it with anything that takes only seconds to compute at most. Feel free to commit to this PR.

File: sktime/forecasting/compose/_multiplexer.py [hunk 1]
```diff
@@ -64,14 +64,14 @@ class MultiplexForecaster(_HeterogenousEnsembleForecaster):
     ...     ForecastingGridSearchCV,
     ...     ExpandingWindowSplitter,
     ...     load_airline)
-    >>> y = load_airline()
+    >>> y = load_airline()[0:24]
```
Comment (aiwalter) [inline sktime/forecasting/compose/_multiplexer.py:67]:
```suggestion
    >>> y = load_shampoo_sales()
```

File: sktime/forecasting/compose/_multiplexer.py [hunk 2]
```diff
@@ -64,14 +64,14 @@ class MultiplexForecaster(_HeterogenousEnsembleForecaster):
     ...     ForecastingGridSearchCV,
     ...     ExpandingWindowSplitter,
     ...     load_airline)
```
Comment (aiwalter) [inline sktime/forecasting/compose/_multiplexer.py:66]:
```suggestion
    ...     load_shampoo_sales)
```

**Review by aiwalter — APPROVED** [review]

**Pull request merged**